src/index.html

```diff
@@ -3,11 +3,65 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+  
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+
+    </div>
+    <div class="stars stars--1">
```

Comment:
add empty lines between multiline sibling blocks and not for parent and child (checklist - point 1)
```suggestion
    </div>
    
    <div class="stars stars--1">
```

Reply:
Thank you. I've made some changes. Please check them